vignettes/projecting_incidence.Rmd

```diff
@@ -291,6 +282,18 @@ We will now plot the individual simulation results alongside the median
 of the aggregated results.
 ```{r viz, fig.cap ="COVID-19 incidence in South Africa projected over a two week window in 2020. The light gray lines represent the individual simulations, the red line represents the median daily cases across all simulations, the black connected dots represent the observed data, and the dashed vertical line marks the beginning of the projection.", fig.width=6.0, fig.height=6}
 
+## since all simulations may end at a different date, we will find the minimum
+## final date for all simulations for the purposes of visualisation
+final_date <- incidence_ts_by_date %>%
+  group_by(sim) %>%
+  summarise(final_date = max(date), .groups = "drop") %>%
+  summarise(min_final_date = min(final_date)) %>%
+  pull(min_final_date)
+incidence_ts_by_date <- incidence_ts_by_date %>%
+  filter(date <= final_date)
+median_daily_cases <- median_daily_cases %>%
+  filter(date <= final_date)
+
```

Comment:
added this to replace the manual axis limits below.

Reply:
Awesome. Thanks.